src/Exceptions/AuthenticationException.php

```diff
@@ -11,6 +11,6 @@ public function toResponse($request)
     {
         return $request->expectsJson()
             ? response()->json(['message' => $this->getMessage()], 401)
-            : redirect()->route('statamic.cp.login');
+            : (config('statamic.cp.auth', true) ? redirect()->route('statamic.cp.login') : abort(401));
```

Comment:
Maybe allow an alternative route to be specified, eg if you want to direct them to the Nova logic page?

Reply:
Good idea! That's possible now.